changelogs/changelog.yaml

```diff
@@ -82,9 +82,25 @@ releases:
         namespace: null
     release_date: '2020-06-23'
   1.0.1:
+    changes: {}
+    fragments: []
+    release_date: '2020-06-23'
+  1.0.4:
     changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in iosxr_config work when module
+        alias is used (https://github.com/ansible-collections/cisco.iosxr/pull/63).
+      - Makes sure that docstring and argspec are in sync and removes sanity ignores
+        (https://github.com/ansible-collections/cisco.iosxr/pull/62).
+      - Update docs after sanity fixes to modules.
       minor_changes:
-      - Bring plugin table to correct position (https://github.com/ansible-collections/cisco.iosxr/pull/44)
+      - Add changelogs for previous releases (https://github.com/ansible-collections/cisco.iosxr/pull/47)
```

Comment:
Are these not supposed to be attached to some earlier release?

Reply:
For that matter, there should probably be some mention of versions 1.0.2 and 1.0.3, even if there aren't any fragments for them besides a release_summary

Reply:
I'll update for the required version